css/style.css

```diff
@@ -0,0 +1,148 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    margin-top: 5rem;
+    background-color: #eaeaec;
+}
+
+/** Navigation bar **/
+.nav-box-bar.desktop-nav {
+    position: fixed;
+    top: 0;
+    background-color: white;
+    padding: 1rem 5rem;
+}
+
+.logo-text {
+    font-size: 1.5rem;
+    font-weight: 900;
+    font-family: Arial, Helvetica, sans-serif;
+    margin-top: 0.30rem;
+}
+
+.logo-text span {
+    color: #ff3b30;
+}
+
+#social-links-bar .no-bg.hov.btn-solid-primary {
+    height: auto;
+    width: auto;
+    background-color: #ff3b30;
+    color: white;
+    margin: 0;
+    margin-right: 1.5rem;
+}
+
+#social-links-bar .no-bg.user-icon {
+    font-size: 1.6rem;
+    height: auto;
+}
+
+#social-links-bar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+}
+/** End of Navigation bar **/
+
+/** Banner section **/
+.banner-section {
+    margin: 1rem 5rem 0rem 5rem;

```

Comment:
just check...whether it is possible to achieve same effect by using 3 values as 1 5 0